dags/populate_metabase_fluxiae.py

```diff
@@ -38,7 +38,18 @@ def import_and_decrypt(import_directory, **kwargs):
 
         with tempfile.NamedTemporaryFile(dir=import_directory) as file:
             most_recent_import = max(obj["Key"] for obj in response["Contents"])
+            kwargs["ti"].xcom_push("imported_file_key", most_recent_import)
```

Comment:
Je trouve plus élégant de faire comme ceci :
```suggestion
            task_instance.xcom_push("imported_file_key", most_recent_import)
```
Pour avoir `task_instance` tu peux regarder ce qui est fait dans `create_work_directory()`

---

Je remplacerais la gestion manuelle XCom par faire retourner `import_and_decrypt()` la valeur de `most_recent_import` plutôt que `import_directory`, j'ai fait ça pour imbriquer les appels tout en bas mais ça me semble pas déconnant que `import_and_decrypt()` retourne le fichier importé plutôt que le dossier passé en paramètre, ainsi on pourrais le passer avec `work_directory` à `process()` et ça serais encore plus proche de ce qu'on aurais fait dans du code python classique, et en plus ça dit à Airflow dans quel sens les tâches doivent être exécutées :)